src/pages/components/action-half-modal.mdx

```diff
@@ -6,6 +6,19 @@ layout: '@layouts/ComponentLayout.astro'
 description: '画面の下部を占有する、アクションを実行するためのモーダルダイアログ'
 thumbnail: '/assets/images/components/thumbnail-action-half-modal.svg'
 repositoryUrl: 'https://github.com/ubie-oss/ubie-ui/tree/main/src/components/ActionHalfModal'
+props:
```

Comment:
Props such as complex union types cannot be automatically extracted by the library, so they are described manually.